README.md

```diff
@@ -374,10 +374,10 @@ python train.py -m datamodule.batch_size=32,64,128 model.lr=0.001,0.0005
 ```bash
 # this will run hyperparameter search defined in `configs/hparams_search/mnist_optuna.yaml`
```

Comment:
```suggestion
# this will run hyperparameter search defined in `configs/hparams_search/conll2003_optuna.yaml`
```

Reply:
fixed in [7ab0710](https://github.com/ChristophAlt/pytorch-ie-hydra-template/pull/42/commits/7ab0710af544c1c9b38236a7aff0f9badb2c0a0a)